manifest.json

```diff
@@ -227,5 +239,62 @@
       "label": "follow_topic_label",
       "value": true
     }]
+  },{
+    "label": "footer_section_contents",
+    "variables": [{
+      "identifier": "show_contact_us_section",
+      "type": "checkbox",
+      "description": "show_contact_us_section_description",
+      "label": "show_contact_us_section_label",
+      "value": true
+    }, {
+      "identifier": "contact_us_header",
+      "type": "text",
+      "description": "contact_us_header_description",
+      "label": "contact_us_header_label",
+      "value": "Can't find what you're looking for?"
+    }, {
+      "identifier": "mit_address",
+      "type": "text",
+      "description": "mit_address_description",
+      "label": "mit_address_label",
+      "value": "Massachusetts Institute of Technology 77 Massachusetts Avenue Cambridge, MA 02139"
+    }, {
+      "identifier": "mit_phone_number",
+      "type": "text",
+      "description": "mit_phone_number_description",
+      "label": "mit_phone_number_label",
+      "value": "617-253-1000"
+    }, {
+      "identifier": "mit_home",
+      "type": "text",
+      "description": "mit_home_description",
+      "label": "mit_home_label",
+      "value": ""
+    }, {
+      "identifier": "mit_about_us",
+      "type": "text",
+      "description": "mit_about_us_description",
+      "label": "mit_about_us_label",
+      "value": ""
+    }, {
+      "identifier": "mit_tos",
+      "type": "text",
+      "description": "mit_tos_description",
+      "label": "mit_tos_label",
+      "value": ""
+    }, {
+      "identifier": "mit_privacy_policy",
+      "type": "text",
+      "description": "mit_privacy_policy_description",
+      "label": "mit_privacy_policy_label",
+      "value": ""
+    }, {
+      "identifier": "mit_honor_code",
+      "type": "text",
+      "description": "mit_honor_code_description",
+      "label": "mit_honor_code_label",
+      "value": ""
```

Comment:
@pdpinch any default ones for these empty values, please mention so I can update it here.